### DISCUSSION TITLE
Welcome to aiz-basic-plus Discussions!

### DISCUSSION THREAD

**Discussion by xaxa-0x3F** [ehAsuna/ais-basic-plus#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

The AizBasic+ community is a friendly community. Please be encouraging towards others and email me if you need help with any issues. codeasaurousteam@gmail.com is my email.

**Comment by witherBattler** [comment]
hello
